alpha/apps/kaltura/lib/kMrssManager.php

```diff
@@ -460,6 +462,17 @@ public static function getEntryMrssXml(entry $entry, SimpleXMLElement $mrss = nu
 		}
 			
 		$categories = explode(',', $entry->getCategories());
+		if (count($features) && in_array (ObjectFeatureType::CATEGORY_ENTRIES, $features))
+		{
+			$categoryEntries = categoryEntryPeer::retrieveActiveByEntryId($entry->getId());
+			$categories = array ();
+			foreach ($categoryEntries as $categoryEntry)
+			{
+				$category = categoryPeer::retrieveByPK($categoryEntry->getCategoryId());
```

Comment:
This is very inefficient, please collect all ids and query them all together using retrieveByPKs.
Also, please disable all entitlement filters to make sure you actually get all the ids.